src/EpisodeManagementService/CreateUpdateEpisode/Program.cs

```diff
@@ -1,12 +1,19 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using NHS.ServiceInsights.Common;
+using Azure.Messaging.EventGrid;
 
 var host = new HostBuilder()
     .ConfigureFunctionsWorkerDefaults()
     .ConfigureServices(services =>
     {
         services.AddSingleton<IHttpRequestService, HttpRequestService>();
+
+        services.AddSingleton(sp =>
+        {
+            var credentials = new Azure.AzureKeyCredential(Environment.GetEnvironmentVariable("topicKey"));
```

Comment:
We only want to use topic key for local development. We should add some conditional logic that checks the environment and uses AzureKeyCredential for Development and DefaultAzureCredential when not.

Reply:
done this.